### ISSUE TITLE
Unable to post or publish.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#551]
```
What's the problem?
After completing a blog, I can't post or publish it to my WordPress.




What browser are you using?
FireFox 8 or the newest 9. There was an update so I fairly certain its 9




What version of ScribeFire are you running?
The latest for FireFox.
```

-----
Original issue reported on code.google.com by `soundsof...@knology.net` on 17 Jan 2012 at 7:49

**Comment by GoogleCodeExporter** [comment]
```
Once I complete a blog, how do I post or publish it?
```

-----
Original comment by `soundsof...@knology.net` on 17 Jan 2012 at 7:50

**Comment by GoogleCodeExporter** [comment]
```
Fixed for blogger blogspot in version 4.3.1
```

-----
Original comment by `scribefi...@gmail.com` on 14 Apr 2014 at 7:08
* Changed state: **Fixed**
